src/interfaces/ICSModule.sol

```diff
@@ -479,6 +488,20 @@ interface ICSModule is IQueueLib, INOAddresses, IAssetRecovererLib {
         bool isSlashed
     ) external;
 
+    /// @notice Report Node Operator's key as withdrawn and settle withdrawn amount
+    /// @notice Called by the CSStrikes contract.
+    ///         See `CSStrikes.processValidatorEjection` to use this method permissionless
+    /// @param nodeOperatorId ID of the Node Operator
+    /// @param keyIndex Index of the withdrawn key in the Node Operator's keys storage
+    /// @param strikesData Strikes of the Node Operator
```

Comment:
it's about node operator keys i believe
it's not clear what are the values supposed to be in this list

Reply:
TODO added, because we don't know what values will be there yet